### PR TITLE
Add example to setup URL rewriting

### PR DESCRIPTION
**Pull request by mrkschan**

### CODE REVIEW

File: src/configuration/app-containers.md [hunk 1]
```diff
@@ -478,6 +478,22 @@ It has a few subkeys which are:
 
 The minimum interval between cron runs is 5 minutes, even if specified as less.
 
+### URL rewrite
+
+Platform.sh supports using URL rewrite in your application. To do so, you have to add a `rules` block to a location.
```
Comment (pjcdawkins) [inline src/configuration/app-containers.md]: "using URL rewriting"

File: src/configuration/app-containers.md [hunk 2]
```diff
@@ -478,6 +478,22 @@ It has a few subkeys which are:
 
 The minimum interval between cron runs is 5 minutes, even if specified as less.
 
+### URL rewrite
```
Comment (pjcdawkins) [inline src/configuration/app-containers.md]: URL rewriting

File: src/configuration/app-containers.md [hunk 3]
```diff
@@ -478,6 +478,22 @@ It has a few subkeys which are:
...
+```
+web:
+  locations:
+    "/":
+      root: "public"
+      passthru: "/index.php"
+        rules:
+          ^/blog/(?<bid>.*)$:
+            passthru: /blog.php?bid=$bid
+```
```
Comment (pjcdawkins) [inline src/configuration/app-containers.md]: With consistent quoting:

```yaml
web:
    locations:
        '/':
            root: 'public'
            passthru: '/index.php'
            rules:
                '^/blog/(?<bid>.*)$':
                    passthru: '/blog.php?bid=$bid'
```

**Comment by mrkschan** [comment]
How could I mark this as "blocked" ... This URL rewrite is not working in py3.5 container.

**Comment by mrkschan** [comment]
updated :)

**Pull request merged**